galsim/photon_array.py

```diff
@@ -205,7 +205,13 @@ def _fromArrays(
         ret._pupil_u = pupil_u
         ret._pupil_v = pupil_v
         ret._time = time
-        ret._is_corr = is_corr
+        ret._is_corr = False
+        if is_corr:
+            from .deprecated import depr
+            depr('is_corr=True', 2.5, '',
```

Comment:
What was the former use case and why is it no longer necessary?

Reply:
It was for the Sum._shoot implementation.  That had done all the photons in one component first, then all the ones in the other component.  I switched that to do the same algorithm I used for ChromaticSum.  This is potentially important if the PSF is time dependent for instance (e.g. includes a PhasePSF), so the order in the PhotonArray matters.  It seems like there are subtle implications to letting them be correlated like that, and it was very easy to switch it to not ever be correlated in the first place.